patterns/direct_processor.py

```diff
@@ -0,0 +1,142 @@
+## Goal: A simpe request pulblisher to publish a request with reply_to set to any topic.
+import os
+import platform
+import time
+
+# Import Solace Python  API modules from the solace package
+from solace.messaging.messaging_service import MessagingService, ReconnectionListener, ReconnectionAttemptListener, \
+                                                ServiceInterruptionListener, RetryStrategy, ServiceEvent
+from solace.messaging.resources.topic import Topic
+from solace.messaging.publisher.direct_message_publisher import PublishFailureListener, FailedPublishEvent
+from solace.messaging.resources.topic_subscription import TopicSubscription
+from solace.messaging.receiver.message_receiver import MessageHandler, InboundMessage
+
+
+if platform.uname().system == 'Windows': os.environ["PYTHONUNBUFFERED"] = "1" # Disable stdout buffer 
+
+# Goal is to demonstrate receive-process-reply (processor) pattern. Processor subscribes to a request topic and upon receipt of 
+# request message carrying a string, the processor converts the payload to uppercase (for simplicity) and publishes to a 
+# designated topic.
+#   direct_requestor: 
+#       subscribes to request topic 'solace/samples/python/direct/processor/input/{count}
```

Comment:
`subscribes to request topic 'solace/samples/python/direct/processor/input/>`

Reply:
Changed the requested topic without any wildcards as there is no function demonstrated on the topic variable.

subscribes to request topic  'solace/samples/python/direct/processor/input'